src/main/java/eu/europeana/metis/sandbox/service/workflow/ExternalValidationServiceImpl.java

```diff
@@ -1,14 +1,30 @@
 package eu.europeana.metis.sandbox.service.workflow;
 
-import eu.europeana.metis.sandbox.common.Step;
+import eu.europeana.metis.sandbox.common.exception.RecordProcessingException;
 import eu.europeana.metis.sandbox.domain.Record;
+import eu.europeana.metis.transformation.service.TransformationException;
 import org.springframework.stereotype.Service;
 
 @Service
 class ExternalValidationServiceImpl implements ExternalValidationService {
 
+  private OrderingService orderingService;
+
+  public ExternalValidationServiceImpl(
+      OrderingService orderingService) {
+    this.orderingService = orderingService;
+  }
+
   @Override
   public Record validate(Record record) {
-    return Record.from(record, Step.VALIDATE_EXTERNAL);
+    String recordOrdered;
+    try {
+      recordOrdered = orderingService.performOrdering(record.getContent());
+    } catch (TransformationException e) {
+      throw new RecordProcessingException(record.getRecordId(), e);
+    }
+
+    return Record.from(record, recordOrdered);
```

Comment:
I assume that validation is to be inserted here at a later stage?

Reply:
yes